### PR TITLE
Solved issue with Box-Shadow in Nav-toggler

### PR DESCRIPTION
**Pull request by AshuKV**
## What is the change?

Redesigned Box-Shadow in Nav-toggler

## Related issue?
close: #879 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
 
image

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -517,6 +517,11 @@ input[name='perPage'] {
 	background: #129fb4;
 }
 
+.nav-toggle-button:hover {
+	box-shadow: 10px 5px 5px rgb(97, 93, 93);
+	opacity: 0.1;
```
Comment (Pranav016) [inline public/css/styles.css:522]: Opacity is making the toggler button look very light (in the screenshot you added).

It looked pretty decent before
image

i think adding only the box-shadow would do the job

**Review by Pranav016 — CHANGES_REQUESTED** [review]
There are changes done to the package-lock file which are not required. Revert those changes.
Check README for instructions for package-lock file version 2 to avoid these changes in the future.

**Review by AshuKV — COMMENTED** [review]
Can you please suggest me what changes I have to do in package-lock.json file.

**Comment by Pranav016** [comment]
> Can you please suggest me what changes I have to do in package-lock.json file.

You have to revert the changes you did in package-lock file.

**Comment by AshuKV** [comment]
The conflicts have been resolved. Finally you could merge the PR now. Thanks for the help.

**Comment by Pranav016** [comment]
> The conflicts have been resolved. Finally you could merge the PR now. Thanks for the help.

There are still changes to the package-lock file which should be reverted.

**Comment by Pranav016** [comment]
You can take help from this https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**